dbt_copilot_helper/commands/application.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+
+import time
+from datetime import datetime
+from datetime import timedelta
+
+import click
+
+from dbt_copilot_helper.utils.aws import get_aws_session_or_abort
+from dbt_copilot_helper.utils.click import ClickDocOptGroup
+from dbt_copilot_helper.utils.versioning import (
+    check_copilot_helper_version_needs_update,
+)
+
+
+def get_query_results(env, app, profile, query_string):
+    project_session = get_aws_session_or_abort(profile)
+
+    click.secho(
+        f"Showing status for app {app} in aws account {profile}",
+        fg="green",
+    )
+    logs_client = project_session.client("logs")
+    ecs_client = project_session.client("ecs")
+
+    response = ecs_client.list_clusters()
+
+    # Find clusters log-group
+    for cluster in response["clusterArns"]:
+        cluster_name = cluster.split("/")[-1]
+        APP = cluster_name.split("-")[0]
+        ENV = cluster_name.split("-")[1]
+        if app == APP and env == ENV:
+            log_group_name = f"/aws/ecs/containerinsights/{cluster_name}/performance"
+            break
+
+    # Container stats are 5 mins behind realtime.
+    date_time = datetime.now()
+    end_time = int(date_time.timestamp())
+    start_time = int((date_time - timedelta(minutes=5)).timestamp())
+    end_time = int((date_time - timedelta(minutes=4)).timestamp())
+
+    click.echo(
+        click.style(
+            "Date & Time:  ",
+            fg="cyan",
+        )
+        + click.style(f"{datetime.utcfromtimestamp(start_time)}", fg="cyan", bold=True)
+    )
+
+    # First you need to create a log query, then retrieve the log query.
+    cpu_response_id = logs_client.start_query(
+        logGroupName=log_group_name,
+        startTime=start_time,
+        endTime=end_time,
+        queryString=query_string,
+    )
+
+    # Need to wait approx 5 seconds for the query to be available.
+    click.secho(
+        "waiting 5s...",
+        fg="cyan",
+    )
+    time.sleep(5)
+
+    cpu_response = logs_client.get_query_results(queryId=cpu_response_id["queryId"])
+
+    return cpu_response
+
+
+@click.group(chain=True, cls=ClickDocOptGroup)
+def application():
+    """AWS Copilot svc actions with DBT extras."""
+    check_copilot_helper_version_needs_update()
+
+
+@application.command()
+@click.option("--env", type=str, required=True)
+@click.option("--app", type=str, required=True)
+@click.option("--project-profile", type=str, required=True)
+@click.option("--storage", is_flag=True)
+@click.option("--network", is_flag=True)
+def stats_long(env, app, project_profile, storage, network):
```

Comment:
Wondering about the utility of stats_long vs stats, is this more a command line flag, a la `ls -l`

Reply:
I guess this depends on peoples preference.  Last time I did something similar the recommendation was to split it out to a seperate command.  The command is different, once is at the task level and the other is container level.  But happy to change if thats the prefered option.

Reply:
Ah, ok, that makes sense, then maybe `stats-long` isn't quite right? To me that means show me the unabbreviated version of the stats, not show me the container stats.

How about something like `copilot-helper application container stats` and `copilot-helper application task stats`.

Also worth noting we have a UR survey out that asks what this command should look like:

> 5. What command would you expect to run, to see resource utilisation and health for your application?

Reply:
Do we have any initial results from that question that might inform us here?

Reply:
I'm posting it to devs channel in a few minutes, so we should get some results soon, though will likely need to wait on the survey end and analysis.

Reply:
yes I think renaming the command stats-long to container is less confusing.

Reply:
renamed to application container-stats/task-stats